elastic/kibana
# [Fleet] Use user configured download source in enrollment instructions

## Summary

Resolve https://github.com/elastic/kibana/issues/164475

Display the configured download source for the agent policy, instead of an hardcoded one.

## UI Change

<img width="779" alt="Screenshot 2024-03-05 at 3 25 34 PM" src="https://github.com/elastic/kibana/assets/1336873/1c2ce1db-5b53-4e50-aeb1-b29a4d84c4f5">
<img width="1230" alt="Screenshot 2024-03-06 at 1 33 03 PM" src="https://github.com/elastic/kibana/assets/1336873/1d471e47-14fc-42eb-b89a-759a0018caba">


## Tests

Updated related unit tests

#### Manual test

you can create a download source ( a default one or one assigned to a policy) and check it's display in the agent enrolment instructions
